netlify.toml

```diff
@@ -1166,4 +1166,9 @@ status = 301
 [[redirects]]
 from = "/testing-distribution/apple-devices"
 to = "/signing-identities/apple-devices"
-status = 301
\ No newline at end of file
+status = 301
+
+[[redirects]]
+from = "/workflows/common-workflow-steps/custom-script#how-to-change-java-version"
```

Comment:
As I remember, Çelik sad that we can remove the example custom script in the faq of custom script. I test to redirection of this URL, I still go to old java version change cs example under CS Faq section.

If I am wrong, please let me know

Reply:
> As I remember, Çelik sad that we can remove the example custom script in the faq of custom script.
right, I thought I removed it.

>  I test to redirection of this URL, I still go to old java version change cs example under CS Faq section.
It doesn't redirect even though I uninstall it.

Reply:
Although the custom script example removed from Custom Scrip FAQ section, this link does not redirect to new component link. I tried with this link @csonuryilmaz 

https://deploy-preview-717--reverent-galileo-8ef035.netlify.app/workflows/common-workflow-steps/custom-script#how-to-change-java-version

Reply:
I had worked on the Netlify redirects about a year ago, so in order to identify the case here, I reworked the Netlify documents to remember the details.

Unfortunately, I saw that Netlify redirects have not been maintained well during this period, which can lead to weird cases later in the future.

👇 First of all, **anybody who will touch this file** should work on the below pages to understand the working principles that affect the redirection.

- https://docs.netlify.com/routing/redirects/
  - https://docs.netlify.com/routing/redirects/redirect-options/

When you work on these resources, you will see the reason behind the below "custom-script" redirection.

![Screenshot 2024-10-27 at 09 40 57](https://github.com/user-attachments/assets/06b6e65a-eef5-4322-a41b-da09f2ffcdd8)

I will not dive into those details and focus on the case here.

**TLDR;** `#` redirection is not supported in Netlify redirects.  You cannot use them in `from` section since the Netlify redirector does not see that part when a request comes it. (**[ref](https://answers.netlify.com/t/redirecting-from-invite-token-to-admin-invite-token/51661)**)

Let me explain the situation with the samples below:

![Screenshot 2024-10-27 at 09 48 45](https://github.com/user-attachments/assets/b7c0fd5d-3a0d-4db0-b91c-5e2bd4dd11ce)

You can use `#` in the `to` section for a redirect rule (_it works_), but `from` section is useless.

We have two options here;

- We can keep that section in the document and add an info box there that explains that the relevant content is moved or changed and redirects the use to the new section.
- OR, we can work on Netlify's (or Docusaurus') other mechanisms to achieve the goal here, which can take more effort to discover other possibilities.

Reply:
@csonuryilmaz As far as I understand, it makes more sense in the short term to keep the document for the time being and to provide guidance with an information message. If yes, @ftmzhrsckldr , could you please take back the sample code we provided under Custom script and reference the new component by redirecting to it?